File: static/skeletons/DnaSequences.java
import java.util.*;
import java.io.*;

class DnaSequences {

	public String evaluate(List<String> sequences) {
		// TODO
		return "";
	}

	/**
	 * Read DNA sequences from standard in, one per line.
	 *
	 * Write to standard out the longest subsequence that any two
	 * sequences have in common.
	 *
	 */
	public static void main(String args[]) throws IOException {
		BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
		List<String> sequences = new ArrayList<String>();
		String line;
		while((line = reader.readLine()) != null) {
			sequences.add(line);
		}

		String result = new DnaSequences().evaluate(sequences);
		System.out.println(result);
	}

}

File: static/skeletons/dna_sequences.py
import sys

def evaluate(sequences):
    #TODO
    pass


def main():
    """Read DNA sequences from standard in, one per line.

    Write to standard out the longest subsequence that any two
    sequences have in common.

    """
    sequences = [line.strip() for line in sys.stdin.readlines()]
    print(evaluate(sequences))


if __name__=='__main__':
    main()
